test/edit-column.html

```diff
@@ -544,6 +544,25 @@
         });
       });
 
+      describe('disable editing', () => {
+
+        let grid, input;
```

Comment:
`input` is declared in the "describe" block but it's not assigned in "beforeEach". Either assign it in "beforeEach" or remove it and use `const input = ...` in the test case.